components/style/style_adjuster.rs

```diff
@@ -7,8 +7,14 @@
 
 use app_units::Au;
 use properties::{self, ComputedValues, StyleBuilder};
+#[cfg(feature = "gecko")]
+use properties::longhands::_moz_math_variant::computed_value::T as moz_math_variant;
 use properties::longhands::display::computed_value::T as display;
 use properties::longhands::float::computed_value::T as float;
+#[cfg(feature = "gecko")]
+use properties::longhands::font_style::computed_value::T as font_style;
+#[cfg(feature = "gecko")]
+use properties::longhands::font_weight::computed_value::T as font_weight;
```

Comment:
nit: when the stuff we use is conditionally compiled, if it's only used inside a function, I tend to just move it inside the function (see `adjust_for_text_combine_upright` for example).

Not a big deal in any case, it's fine to file an E-easy so it gets cleaned up in a followup.